test/test_4_cell_cell_communication.py

```diff
@@ -123,7 +121,7 @@ def test_odds_ratio_no_match():
     openproblems.log.debug(
         "Testing {} metric from {} task".format(metric.__name__, task.__name__)
     )
-    adata.uns["ccc_target"]["response"] = np.nan
+    adata.uns["ccc_target"]["response"] = 0
     m = metric(adata)
 
-    assert m == 1
+    assert m is np.inf
```

Comment:
Makes it more obvious that an issue was encountered - any meaningful scenarios are captured by the other ifs.

Reply:
Much better!